frontend/src/pages/CoursePage.vue

```diff
@@ -1,250 +1,281 @@
 <template>
-	<!--Content wrapper-->
```

Comment:
This is all commiting to using wrong indent type, with very few lines of code added. Quite hard to see what is actually being added,